README.md

```diff
@@ -256,7 +256,7 @@ The server can be run from the command line. You can either install it globally:
 
 ```shell
 npm install -g oauth2-mock-server
-oauth2-mock-server --help
+npx oauth2-mock-server --help
```

Comment:
You already have this just 5 lines below. 😛

Reply:
>Do you think that we should drop the global install option from the README?

@poveden I do think so

Rationale:
- First line of the doc recommends to install it locally. A npx call will honor this and first search for local packages.
- In the other case, npx will download and cache the package. Which in my mind looks very similar to a global install. With an additional plus of getting the latest version.
- Power users who are used to globally installing packages and are willing to keep on doing it don't need a line in this documentation :wink: